staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -60,6 +60,16 @@ var (
 		allAlphaGate: setUnsetAlphaGates,
 		allBetaGate:  setUnsetBetaGates,
 	}
+
+	// DefaultMutableFeatureGate is a mutable version of DefaultFeatureGate.
+	// Only top-level commands/options setup should make use of this.
+	// Tests that need to modify feature gates for the duration of their test should use:
+	//   defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.<FeatureName>, <value>)()
```

Comment:
This probably should be the new import path.

`hack/verify-test-featuregates.sh` also needs to be updated. Right now it complains:
```
Test files may not access mutable global feature gates directly:
./staging/src/k8s.io/component-base/logs/json/register/validate_test.go:64:			defer featuregatetesting.SetFeatureGateDuringTest(t, featuregate.DefaultMutableFeatureGate, v1.LoggingAlphaOptions, test.alphaEnabled)()
...
Use this invocation instead:
  defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.<FeatureName>, <value>)()
```

Reply:
The script was fine, the code shouldn't have used DefaultMutableFeatureGate. Fixed.